docs/decisions/0008-compatibility-scheme.rst

```diff
@@ -0,0 +1,58 @@
+0008: Compatibility scheme with the current system
+###################################################
+
+Status
+******
+
+**Draft** *2025-09-29*
+
+Context
+*******
+
+Open edX has its authorization system described in the `OEP-66`_, but due to its limitations, the community wanted to explore a more appropriate option for managing authorization on the platform. To mitigate the possible risk associated with completely overhauling a core system like authorization, our primary strategy is to implement a staging or phased migration plan. This approach enables us to limit the blast radius to test components in a controlled environment, apply lessons learned, and ensure business continuity, thereby giving users time to adapt.
+
+Decision
+********
+
+* The new authorization will coexist with the previous one until we migrate the entire system.
+* We will start migrating the current library permissions and roles to the new authorization system.
```

Comment:
What did you find in your investigation about bridgekeeper? I made some decisions [here](https://openedx.atlassian.net/wiki/spaces/OEPM/pages/5210112002/Open+edX+AuthZ+Framework+Long-Term+Vision#What-about-Bridgekeeper?), but I want to understand what this would look like in practice during the migration.